### ISSUE TITLE
Missing argument in MNIST example

### ISSUE THREAD

**Issue by danperazzo** [pytorch/opacus#517]
## 📚 Documentation

Hello everyone, I have been reading the [MNIST_README.md](https://github.com/pytorch/opacus/blob/main/examples/mnist_README.md) to run its example and I noticed an issue.

It mentions that, to run it, we have to use:
```shell
python mnist.py --device=cpu -n=15 --lr=.25 --sigma=1.3 -c=1.5 -sr=0.004
```
However, there is no argument `sr` on the code, and the results are too different from the reported ones.
Thanks for the attention!

**Comment by pierrestock** [comment]
Hey danperazzo,

Thanks for flagging this! Labelling is at bug, will investigate.

**Comment by xuefeng-xu** [comment]
'-sr' means poisson sampling rate, but the code use '-b' batch size, so just change -sr=0.004 to -b=512, for example.

**Comment by ffuuugor** [comment]
Fixed with #520
